content/tutorial/02-sveltekit/02-routing/03-loading-data/README.md

```diff
@@ -0,0 +1,36 @@
+---
+title: Loading Data
+---
+
+So far, we have seen how to create pages and layouts with static UI data. Most of your apps need to load some kind of data however. You could do this with a regular `fetch` inside `onMount`. That doesn't work with server-side-rendering or prerendering though, and you'd have half-finished pages loading data only after they are rendered.
+
+SvelteKit provides a better way to do this through `load` functions.
+
+This `load` function can appear in several places. Let's start with the `load` function inside `+page.js`.
```

Comment:
```suggestion
This `load` function can appear in several places. Let's start with adding the `load` function inside `+page.js`.
```

Reply:
To me the current sentence feels more correct - it's not about the first step of what to do in this specific tutorial section but rather that we start with the load function inside this file.